README.md

```diff
@@ -394,10 +396,57 @@ The `extract` method takes no arguments:
     1. `v` must have the same type that `f` returns in `extend`.
 
 
+### Bifunctor
 
+A value that implements the Bifunctor specification is a functor with two
+type arguments that are both covariant.
```

Comment:
I think it is necessary two indicate that there are two type variables that are modified by `bimap`. We could potentially drop the mention of covariance though.